arch/riscv/mach-duowen/gpio.c

```diff
@@ -54,6 +54,14 @@ struct duowen_pad duowen_pads[] = {
 	{ GPIO2A, pad_gpio_108, TLMM_PAD_FUNCTION, GPIO_PAD_PULL_UP, 8 },
 	{ GPIO2A, pad_gpio_109, TLMM_PAD_FUNCTION, GPIO_PAD_PULL_UP, 8 },
 #endif
+#ifndef CONFIG_DUOWEN_FLASH_SPI0
```

Comment:
Should it be CONFIG_DUOWEN_SSI_FLASH_SPI0?

Reply:
fixed